<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>打开文件</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="cifa">
      <property name="text">
       <string>词法分析</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QTextEdit" name="textEdit_2"/>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="cifa_save">
      <property name="text">
       <string>词法分析保存</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="yufa">
      <property name="text">
       <string>语法分析</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QTextEdit" name="textEdit_4"/>
    </item>
    <item row="3" column="2" colspan="2">
     <widget class="QTextEdit" name="textEdit_3">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="6" column="2" colspan="2">
     <widget class="QTextEdit" name="textEdit_5"/>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="yufa_show">
      <property name="text">
       <string>展示语法分析</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>语义分析</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2" colspan="2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>中间代码生成</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
